run.NGD_v3atm.F1850.sh

```diff
@@ -0,0 +1,659 @@
+#!/bin/bash -fe
+
+# E3SM Water Cycle v2 run_e3sm script template.
+#
+# Inspired by v1 run_e3sm script as well as SCREAM group simplified run script.
+#
+# Bash coding style inspired by:
+# http://kfirlavi.herokuapp.com/blog/2012/11/14/defensive-bash-programming
+
+main() {
+
+# For debugging, uncomment libe below
+#set -x
+
+# --- Configuration flags ----
+
+# Machine and project
+readonly MACHINE=chrysalis
+readonly PROJECT="e3sm"
+
+# Simulation
+readonly COMPSET="F1850_chemUCI-Linozv3"
+readonly RESOLUTION="ne30pg2_EC30to60E2r2"
+readonly CASE_NAME="20230223.NGD_v3atm.F1850"  
+
+# Code and compilation
+readonly CHECKOUT="20230223"
+readonly BRANCH="NGD_v3atm_v2.1ocnice_alt" 
+readonly CHERRY=( )
+readonly DEBUG_COMPILE=false
+
+# Run options
+readonly MODEL_START_TYPE="initial"  # 'initial', 'continue', 'branch', 'hybrid'
+readonly START_DATE="0001-01-01"
+
+# Additional options for 'branch' and 'hybrid'
+readonly GET_REFCASE=TRUE
+readonly RUN_REFDIR="/lcrc/group/e3sm/ac.mwu/archive/20220504.v2.LR.bi-grid.amip.chemMZT.chrysalis/archive/rest/1985-01-01-00000"
+readonly RUN_REFCASE="20220504.v2.LR.bi-grid.amip.chemMZT.chrysalis"
+readonly RUN_REFDATE="1985-01-01"
```

Comment:
As already mentioned, we need to have proper initial file for F1850 (gas species level to be 1850). We might try my previous amip MOZART historical runs

Reply:
Did you previous amip run start from 1850? For now, I just extracted the gas species and append to the eam.i from an existing piControl run. (Not using the reference case in this script).